### PR TITLE
Update Episode 1 merge images to improve accessibility

### PR DESCRIPTION
**Pull request by agreene71**
I've edited the 01-basics.md file to link to updated images for play-changes.svg, versions.svg, and merge.svg (e.g., play-changes_GREENE_CONTRIB_01.svg.  The new images include dash patterns as additional distinguishing features for the colored lines indicating changes to the document in each image.  This was done to improve accessibility, and is based on recommendations from WCAG 2.0.  
Note: I am not sure the images have been linked with correct markup syntax on the edited 01-basics.md file.

### CODE REVIEW

File: _episodes/01-basics.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ think of it as a recording of your progress: you can rewind to start at the base
 document and play back each change you made, eventually arriving at your
 more recent version.
 
-![Changes Are Saved Sequentially]https://github.com/agreene71/git-novice/blob/a-greene-checkout-edit-01/fig/play-changes_GREENE_CONTRIB_01.svg
+![Changes Are Saved Sequentially](https://github.com/agreene71/git-novice/blob/a-greene-checkout-edit-01/fig/play-changes_GREENE_CONTRIB_01.svg)
```
Comment (agreene71) [inline _episodes/01-basics.md:38]: Fixed image markup.

**Comment by kekoziar** [comment]
Hi, @agreene71.  I appreciate the change to improve universal access.  Please change the names of the SVGs file to the original (`play-changes`, `merge`, `versions`), and revert the changes in `01-basics.md` to the original code.  Since this lesson is under version control, if someone wants a previous version of `merge.svg`, they only need to go to a previous version of the repository to access it.

**Comment by agreene71** [comment]
I am out of town until tomorrow and will take care of this as soon as I can
upon my return.

Thanks,
Andrew

**Review by kekoziar — CHANGES_REQUESTED** [review]
Hi, @agreene71.  Are you able to make the file name changes?  If you need assistance, please let me know.  Thanks!

**Comment by agreene71** [comment]
Apologies, yes, if assistance could be provided it would be greatly
appreciated.

Thanks,
Andrew

**Comment by kekoziar** [comment]
Hi, @agreene71.  I think the easiest way to edit the files for your PR is on the web-interface in GitHub.  If you click on the "Files changed" tab of the PR page, you'll see an ellipsis in the top right for each file listed.  Clicking that will show a menu that includes an "Edit file" option.  When you choose "Edit file" it allows you to change the text in the file *and* the file name.  Let me know if this method works for you.

**Comment by agreene71** [comment]
Thank you so much for the guidance.  I think the changes you requested are
done, but please let me know if this is not the case.

Best Regards,
Andrew

**Review by kekoziar — APPROVED** [review]
@agreene71.  Thanks for making the changes.  It looks good.
PR verified and approved.

**Comment by agreene71** [comment]
Great, thanks!

Best,
Andrew

**Pull request merged**